io.c

```diff
@@ -10667,15 +10670,18 @@ rb_fcntl(VALUE io, VALUE req, VALUE arg)
 
 /*
  *  call-seq:
- *     ios.fcntl(integer_cmd, arg)    -> integer
+ *    fcntl(integer_cmd, arg) -> integer
+ *
+ *  Calls system call {fcntl(2)}[https://linux.die.net/man/2/fcntl].
+ *
+ *  Issues a low-level command to an I/O device.
+ *
+ *  If argument +argument+ is an integer, it is passed directly;
+ *  if it is a string, it is interpreted as a binary sequence of bytes.
+ *
+ *  Arguments +integer_cmd+ and +argument+, as well as the returned value,
+ *  are platform-dependent.
  *
- *  Provides a mechanism for issuing low-level commands to control or
- *  query file-oriented I/O streams. Arguments and results are platform
- *  dependent. If <i>arg</i> is a number, its value is passed
- *  directly. If it is a string, it is interpreted as a binary sequence
- *  of bytes (Array#pack might be a useful way to build this string). On
- *  Unix platforms, see <code>fcntl(2)</code> for details.  Not
- *  implemented on all platforms.
```

Comment:
I think this phrases from original docs should be preserved (even if rewritten):
* "...to control or query file-oriented I/O streams"
* "Array#pack might be a useful way to build this string"
* " Not implemented on all platforms."

Reply:
This comment seems not to be addressed

Reply:
Modified.